PostResponses/PostResponsesVDRs.js

```diff
@@ -1,21 +1,30 @@
 if (fromVendor){
-  // This step is needed to ensure that nothing else is executed if you're performing a simple GET request. 
-  done(transformedObject)
+  // This step is needed to ensure that nothing else is executed if you're performing a simple GET request.
+  done(transformedObject);
 }
 
 if (!fromVendor) {
-  // This block is to make sure a POST request is sending data in the right format for the vendor (Hubspot). 
-  var emailProperty = {
-    "property":"email",
-    "value": originalObject.email
-  };
-  var properties = [emailProperty]
-  transformedObject.properties = properties
+  // This block is to make sure a POST request is sending data in the right format for the vendor.
+  // You can customize what is being sent in a POST request here, or add more logic that might be needed for complex payloads.
+  done(transformedObject);
 }
 
-if ((originalObject.hasOwnProperty("properties")) && (fromVendor)) {
-  //Finally, this block will determine if data is coming from the vendor and the response has the properties field (a field unique to POST responses).
-  //This block will run the POST response through the VDR. 
-    transformedObject.email = originalObject.properties.email.value
+if ((originalObject.hasOwnProperty("raw")) && (fromVendor)) {
```

Comment:
@kiawnna I think this if block should be moved to the first in the file and add a done statement inside the block.  For a post call that returns a payload with 'raw', the first and the last if statements are true.  Since there is only one done statement (at the end of the file), any code in both of those blocks will run.

Reply:
@nick-bair agreed. done!